editor/src/components/navigator/dependency-list-item.tsx

```diff
@@ -1,14 +1,18 @@
 /** @jsx jsx */
-import { jsx, keyframes } from '@emotion/core'
+import { jsx, keyframes } from '@emotion/react'
 import * as React from 'react'
 import { Button, colorTheme, FlexRow, FunctionIcons, Icons, UtopiaTheme, Tooltip } from 'uuiui'
 import type { PackageDetails } from './dependency-list'
 import { NpmDependencyVersionAndStatusIndicator } from './dependecy-version-status-indicator'
 import { ContextMenuItem } from '../context-menu-items'
 import { MomentumContextMenu } from '../../uuiui-deps'
-import { MenuProvider } from 'react-contexify'
+import { MenuProvider as ReactContexifyMenuProvider } from 'react-contexify'
 import { NO_OP } from '../../core/shared/utils'
 
+// FIXME: For some reason we've been able to use this incorrectly
+// according to the types, but following the types causes the code to fail.
+const MenuProvider = ReactContexifyMenuProvider as any
```

Comment:
Well that's concerning!

Reply:
Agreed, it's used in 3 places and out of the blue TypeScript started complaining about this one.